client/core/notification.go

```diff
@@ -613,3 +614,18 @@ func newBotNote(topic Topic, subject, details string, severity db.Severity, repo
 		Report:       report,
 	}
 }
+
+// LoginNote is a notification with the recent login status.
+type LoginNote struct {
+	db.Notification
+	Message string `json:"msg"`
+}
+
+const TopicLoginStatus Topic = "LoginStatus"
+
+func newLoginNote(message string) *LoginNote {
+	return &LoginNote{
+		Notification: db.NewNotification(NoteTypeLogin, TopicLoginStatus, "", "", db.Data),
+		Message:      message,
```

Comment:
Doesn't look like we need to expand on the `db.Notification`.

```go
// LoginNote is a notification with the recent login status.
type LoginNote struct {
	db.Notification
}

const TopicLoginStatus Topic = "LoginStatus"

func newLoginNote(message string) *LoginNote {
	return &LoginNote{
		db.NewNotification(NoteTypeLogin, TopicLoginStatus, message, "", db.Data),
	}
}
```

Frontend can just use `CoreNote`.